### ISSUE TITLE
useMeshForwardEvents not working for a plane with non-1 size

### ISSUE THREAD

**Issue by Ledzz** [coconut-xr/xinteraction#1]
Following [forwarding](https://coconut-xr.github.io/xinteraction/#/forwarding) tutorial I've tried to use it for a plane that is not the default size. However, the coordinates of a forwarded event seemed off.
I tried to take a look to the source code, and I guess there's a problem with the [computeIntersection](https://github.com/coconut-xr/xinteraction/blob/main/src/react/forward-events.tsx#L398) function. It doesn't take into account the plane size. I've managed to fix it in my project with yarn patch attached, but I'm not sure if this is the right way to do it.
[@coconut-xr-xinteraction-npm-0.1.12-32a02d7404.patch](https://github.com/coconut-xr/xinteraction/files/15336086/%40coconut-xr-xinteraction-npm-0.1.12-32a02d7404.patch)

**Comment by bbohlender** [comment]
In the rewrite I am currently using the uv coords from the mesh, which should work in more cases. Do you see a reason this wouldn't work for you or see a use case where this fails? :)

**Comment by Ledzz** [comment]
No, using the UV seems like a reasonable implementation for me, thanks 😊
